packages/mdc-select/mdc-select.scss

```diff
@@ -93,12 +88,165 @@
+      display: none;
+    }
+
+    display: flex;
+    position: relative;
+    flex: 1;
+    width: 100%;
+    height: 56px;
+    padding-left: 16px;
+    border: none;
+    border-radius: 4px 4px 0 0;
+    background-color: rgba(black, .04);
+    overflow: hidden;
+
+    &::before,
+    &::after {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      border-radius: 50%;
+      background-color: inherit;
+      content: "";
+      opacity: 0;
+      pointer-events: none;
+    }
+
+    // NOTE: For some reasons, stylelint complains that the two patterns below don't follow BEM.
+    // However, it seems to emit the wrong selector for that pattern. Thus, we disable it above where
+    // we would normally disable it (&.mdc-textfield--disabled) as a workaround.
+
+    // stylelint-disable plugin/selector-bem-pattern
+    // TODO: Refactor this out once #194 lands
+    $opacity-duration: 83ms;
+
+    &::before {
+      transition: opacity 17ms linear, transform 0ms $opacity-duration+2 linear;
+    }
+
+    &:hover {
+      .mdc-select__bottom-line {
+        transform: scaleY(2);
+        background-color: rgba(black, .8);
+      }
+
+      &::before {
+        transition:
+          opacity $opacity-duration linear 17ms,
+          transform 250ms $mdc-animation-deceleration-curve-timing-function 17ms;
+        opacity: 1;
+      }
+    }
+
+    &.mdc-ripple-upgraded {
+      @include mdc-ripple-base;
+      // TODO: Add ::before as ripple bg once #194 lands
+      @include mdc-ripple-fg((pseudo: "::after", opacity: .04));
+
+      &::before {
+        top: calc(50% - var(--mdc-ripple-fg-size, 100%) / 2);
+        left: calc(50% - var(--mdc-ripple-fg-size, 100%) / 2);
+        width: var(--mdc-ripple-fg-size, 100%);
+        height: var(--mdc-ripple-fg-size, 100%);
+        transform: scale(.8);
+        content: "";
+      }
+
+      &:hover::before {
+        transform: scale(calc(var(--mdc-ripple-fg-scale, 1) + .1));
+      }
+      // stylelint-enable
+
+      @include mdc-theme-dark(".mdc-textfield", true) {
+        @include mdc-ripple-fg((pseudo: "::after", base-color: white, opacity: .05));
+      }
+    }
+  }
+
+  &__label {
+    @include mdc-rtl-reflexive-position(left, 16px);
+
+    position: absolute;
+    bottom: 12px;
+    transform-origin: left top;
+    transition: mdc-select-transition(transform);
+    color: rgba(black, .6);
+    pointer-events: none;
+    // Force the label into its own layer to prevent to prevent visible layer promotion adjustments
+    // when the ripple is activated behind it.
+    will-change: transform;
+
+    @include mdc-theme-dark(".mdc-select") {
+      @include mdc-theme-prop(color, text-secondary-on-dark);
+    }
+
+    &--float-above {
+      transform: translateY(-40%) scale(.75, .75);
+    }
+  }
+
   &__selected-text {
+    display: flex;
+    align-items: flex-end;
+    margin-bottom: 6px;
+    padding-right: 10px;
+    padding-left: 16px;
```

Comment:
It this supposed to be indented relative to the `__label`?

![image](https://user-images.githubusercontent.com/409245/29099706-db85c766-7c5b-11e7-9013-a80ca61cc090.png)

It looks a little strange to me, but I'm not sure where the relevant spec is.